infra/scoped/locals.tf

```diff
@@ -6,6 +6,10 @@ locals {
     "ManagedBy"   = var.tag_managed_by
   }
 
+  stage_test_client_ids = [
+    # Developer client ids can be added here
```

Comment:
I think we should probably define the developer clients in TF - better that than either blatting them in an apply or excluding them.